common/v2/features/AddAccount/components/DeterministicWallets.tsx

```diff
@@ -19,6 +19,9 @@ import nextIcon from 'assets/images/next-page-button.svg';
 import prevIcon from 'assets/images/previous-page-button.svg';
 import radio from 'assets/images/radio.svg';
 import radioChecked from 'assets/images/radio-checked.svg';
+import { getNetworkByDPath } from 'v2/services/Store/Network/helpers';
```

Comment:
**convention**: Avoid direct imports from files and import from the service instead `v2/services/Store`.
FYI: we're in the middle of rewrite and want to harmonise our services. As an intermediate step we are:
1. using the explicit exports from the `index.ts` files to create a list of the functions that are used.
2. importing only from the service level to decouple the usage from service implementation and simplify the future refactor.